specification/iotoperations/IoTOperations.Management/models/instance.tsp

```diff
@@ -42,4 +50,67 @@ model InstanceProperties {
   @doc("The Azure IoT Operations version.")
   @visibility("read")
   version?: string;
+
+  @doc("The reference to the Schema Registry for this AIO Instance.")
+  @pattern("^[a-z0-9][a-z0-9-]*[a-z0-9]$")
+  schemaRegistryNamespace: string;
+
+  @doc("The configuration for components of the AIO Instance.")
+  components?: Components = #{};
+}
+
+@doc("The configuration for components of the AIO Instance.")
+model Components {
+  @doc("The properties of the ADR instance.")
+  adr?: AdrProperties = #{ state: OperationalMode.Enabled };
+
+  @doc("The properties of the AKRI instance.")
+  akri?: AkriInstanceProperties = #{ state: OperationalMode.Enabled };
+
+  @doc("The properties of the connectors instance.")
+  connectors?: ConnectorInstanceProperties = #{
+    state: OperationalMode.Enabled,
+  };
+
+  @doc("The properties of the dataflows instance.")
+  dataflows?: DataFlowInstanceProperties = #{ state: OperationalMode.Enabled };
```

Comment:
>DataFlowInstanceProperties [](http://example.com/codeflow?start=14&length=26)

Should it be `DataflowInstanceProperties`? You've changed the casing in this PR but there're a couple in this file with an uppercase `F`.